modules/Hoot/managers/api.js

```diff
@@ -1316,4 +1341,18 @@ export default class API {
             .catch( err => window.console.error( err ) );
     }
 
+    getOverpassStats(url) {
+        // We do this because only the portion after data= should be encoded
+        const splitData = url.split(/(data=)/),
+            encodedUrl = splitData[0] + splitData[1] + encodeURIComponent(splitData[2]);
```

Comment:
When doing it with a regex it keeps the 'data=' part because i want to reuse it again so splitData[1] === 'data='

If i used url.split('data=') it would split in 2 parts but it would remove the 'data=' string which i need for the url

Reply:
Thanks, I learned something :apple: